scripts/README.md

```diff
@@ -12,6 +12,7 @@
 | `lsync.sh` | This script checks if the English version of a page has changed since a localized page has been committed. |
 | `replace-capture.sh` | This script sets K8S_WEBSITE in your env to your docs website root or rely on this script to determine it automatically |
 | `check-ctrlcode.py` | This script finds control-code(0x00-0x1f) in text files. |
+| `ja/verify-spelling.sh` | This script finds Japanese words that are against the guideline. |
```

Comment:
nit: It's not clear what the 'guideline' is referring to.

Reply:
The guideline is https://kubernetes.io/ja/docs/contribute/localization/